src/js/media-container.js

```diff
@@ -30,11 +30,15 @@ template.innerHTML = `
     }
 
     /* Video specific styles */
-    :host(:not([audio])) {
+    :host(:not([audio]):not([aspect-ratio])) {
       height: 480px;
       width: 720px;
     }
 
+    :host([aspect-ratio]) {
+      width: 100%;
```

Comment:
Doesn't this need a height of 0px to work? Does that break the inner layout within the media container?